### PR TITLE
postgres integration improvements

### LINKED ISSUE #218: improvements to postgres integration
we should move this tutorial: https://jupysql.ploomber.io/en/latest/howto/postgres-connect.html under the integrations section

also, remove this from here and migrate it to the posgres tutorial: https://jupysql.ploomber.io/en/latest/api/configuration.html?highlight=pgspecial#postgresql-features

finally, we should document how to install pgspecial: https://github.com/ploomber/jupysql/issues/48#issuecomment-1384506476

### PR DESCRIPTION
**Pull request by gtduncan**
## Describe your changes
1. Moved `postgres-connect` tutorial to `doc/integrations`
2. Migrated pgspecial commands to `postgres-connect`
3. Documented how to install pgspecial

## Issue ticket number and link
Closes #218 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--259.org.readthedocs.build/en/259/

### CODE REVIEW

File: doc/api/configuration.md [hunk 1]
```diff
@@ -187,17 +187,4 @@ INSERT INTO points VALUES (1, 1);
 CREATE TABLE more_points (x, y);
 INSERT INTO more_points VALUES (0, 0);
 INSERT INTO more_points VALUES (1, 1);
-```
-
-## PostgreSQL features
-
```
Comment (neelasha23) [inline doc/api/configuration.md:193]: There are some occurences of `shape: (0, 0)` in this doc. This looks incorrect?

File: doc/integrations/postgres-connect.ipynb [hunk 2]
```diff
@@ -766,6 +767,25 @@
...
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## PostgreSQL features\n",
+    "\n",
+    "For reference, `psql`-style \\\"backslash\\\" [meta-commands](https://www.postgresql.org/docs/9.6/static/app-psql.html#APP-PSQL-META-COMMANDS) commands (``\\\\d``, ``\\\\dt``, etc.) are provided by [PGSpecial](https://pypi.python.org/pypi/pgspecial). Example:"
```
Comment (neelasha23) [inline doc/integrations/postgres-connect.ipynb:778]: This doesn't look very clear. Please add a better example if possible

**Comment by neelasha23** [comment]
Please convert the `integrations/postgres-connect.ipynb` file to `.md` format : `jupytext postgres-connect.ipynb --to myst`

**Comment by gtduncan** [comment]
> Please convert the `integrations/postgres-connect.ipynb` file to `.md` format : `jupytext postgres-connect.ipynb --to myst`

When I worked on this, it was an `.ipynb` file with pre-existing executed outputs and was excluded from running in `conf.py` to preserve the outputs. I'm assuming this is due to the docker code cells needing to have already been run, so if I convert it to a `.md` file, these outputs are lost. Should I still convert it to `.md`? If I exclude it from running in `conf.py` then, no output at all is displayed, and otherwise, it gives the error:

`docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`

**Comment by edublancas** [comment]
you're right @gtduncan: let's keep it as an ipynb to  keep the outputs since we cannot run this in the docs building process

**Review by edublancas — CHANGES_REQUESTED** [review]
A few suggestions:

Rename: Get a PostgreSQL instance -> PostgreSQL

[Rename](https://jupysql--259.org.readthedocs.build/en/259/integrations/postgres-connect.html#postgresql-features): PostgreSQL features -> Executing meta-commands. Also, please add some more detail in such section "in order to run meta-commands, you can install pgspecial ..." and also link to the section in postgres-install where we explain how to install pgspecial, the section must also be moved before the "Clean up" section

**Review by edublancas — APPROVED** [review]

**Pull request merged**